io.github.limo_app.limo.json

```diff
@@ -0,0 +1,293 @@
+{
+  "app-id": "io.github.limo_app.limo",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.15-23.08",
+  "sdk": "org.kde.Sdk",
+  "sdk-extensions" :
+  [
+    "org.freedesktop.Sdk.Extension.rust-stable",
+    "org.freedesktop.Sdk.Extension.llvm16"
+  ],
+  "build-options":
+  {
+     "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin",
+        "env": {
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_X86_64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER": "clang",
+            "CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_RUSTFLAGS": "-C link-arg=-fuse-ld=/usr/lib/sdk/rust-stable/bin/mold"
+        }
+  },
+  "command": "Limo",
+  "finish-args":
+  [
+    "--share=ipc",
+    "--device=dri",
+    "--share=network",
+    "--socket=wayland",
+    "--socket=fallback-x11",
+    "--filesystem=home",
+    "--filesystem=/mnt",
+    "--filesystem=/media",
+    "--filesystem=/run/media",
+    "--talk-name=org.freedesktop.Flatpak",
```

Comment:
> allow users to add launch commands for custom modding tools which are to be run on the host.

What kind of commands are these usually? Do you have examples?

`unrar` can be included in the Flatpak package

Reply:
These are usually commands that run some external tool meant to edit an applications files in a way that cannot be done by simply replacing files. [Here](https://steamcommunity.com/sharedfiles/filedetails/?id=2442018077) is an example guide where one such tool is added to a similar application.
Launching these external tools through this application is technically not required for its function but does provide a convenient way for users to collect them all in one place.